test/e2e/specs/footer/00-footer.spec.js

```diff
@@ -0,0 +1,61 @@
+var chaisePage = require('./../../utils/chaise.page.js');
+var chaiseConfig = {
+    name: "Footer",
+    apps:['record','recordedit','recordset'],
+    defaultTables: {
```

Comment:
These table and schema names might change in the future. Don't rely on outside data. Create your own schema and table for these test cases.